modules/packages/msiexec.bat

```diff
@@ -1,169 +1,169 @@
-@echo off

```

Comment:
Hmm. :( Running `dos2unix` on this file seemed to "clean things up" and may have had something to do with success. Sorry for the big diff. I'll be testing on win12,16,19 in the clouds to ensure all works well. I can split up dos2unix and actual changes into separate commits if need be.

Reply:
Well. Checked out in win12,win16,win19 so I say print it. :) Easier to edit for most of us if this is in unix line endings?